.gitignore

```
@@ -22,3 +28,4 @@
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
 replay_pid*
+/flink-sql-runner/target/
```

```suggestion
target
dependency-reduced-pom.xml
```

When I ran `mvn clean install -DskipTests` I ended up with a dependency reduced pom. Also could be worth broadening it to all `target` dirs so future maven modules are covered.